kayenta-atlas/src/main/java/com/netflix/kayenta/atlas/security/AtlasNamedAccountCredentials.java

```diff
@@ -37,16 +37,19 @@
   @Singular
   private List<Type> supportedTypes;
 
+  @Override
+  public String getType() {
+    return "atlas";
+  }
+
   @NotNull
   private AtlasCredentials credentials;
 
   private String fetchId;
 
-  @Override
-  public String getType() {
-    return "atlas";
-  }
+  private List<String> recommendedLocations;
 
   @JsonIgnore
   private BackendUpdater backendUpdater;
+
```

Comment:
Is this extra blank line intentional? Have seen this with some of the `.scala` files, so maybe it's just how you auto-format.

Reply:
Oops.  It's likely just an editor issue...  I notice sometimes if I save a file while the cursor is on a line with blanks, it will preserve them.